openfl/transport/grpc/client.py

```diff
@@ -162,8 +163,10 @@ def create_insecure_channel(self, uri):
             An insecure gRPC channel object
 
         """
-        self.logger.warn(
-            'gRPC is running on insecure channel with TLS disabled.')
+        if not self.notified_about_insecure:
```

Comment:
I think we can do it without `self.notified_about_insecure` variable,  we can replace 
```Python3
self.logger.warn(
                'gRPC is running on insecure channel with TLS disabled.')
            self.notified_about_insecure = True
```
code inside initialization, before the following line https://github.com/intel/openfl/blob/fd919f449119224290f1237a5700b156b565ba42/openfl/transport/grpc/client.py#L123
Correct me if I am wrong =)

Reply:
I agree, multiple warnings are spammed by reconnect.